src/extension/notifier/src/main/java/au/org/emii/wps/HttpNotifier.java

```diff
@@ -0,0 +1,39 @@
+package au.org.emii.wps;
+
+import java.net.URL;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.geotools.data.ows.HTTPClient;
+import org.apache.http.client.utils.URIBuilder;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public class HttpNotifier {
+    private static final Logger logger = LoggerFactory.getLogger(NotifierProcess.class);
```

Comment:
This logger is unused, it could be removed (if you do remove it there's a few import statements that can go, too)